src/FSharp.Data.GraphQL.Samples.GiraffeServer/HttpHandlers.fs

```diff
@@ -64,15 +64,15 @@ module HttpHandlers =
             printfn "Received query: %s" query
             printfn "Received variables: %A" variables
             let query = query |> removeSpacesAndNewLines
-            let result = Schema.executor.AsyncExecute(query, variables = variables, data = Schema.root) |> Async.RunSynchronously
+            let result = Schema.executor.AsyncExecute(query, variables = variables, data = Schema.root, meta = Schema.executionMetadata) |> Async.RunSynchronously
```

Comment:
Metadata can now be passed on each execution call. In this example, it is fixed, but now it could add per-request information, since this method is being called on an endpoint call.